src/sidebar/test/oauth-auth-test.js

```diff
@@ -372,6 +368,8 @@ describe('sidebar.oauth-auth', function () {
     });
 
     it('persists refreshed tokens to storage', () => {
+      fakeSettings.services = [];
```

Comment:
Previously this setup was inherited from the `beforeEach` in the parent block, but now it needs to be provided with each test case.

Setup "inheritance" is one reason why wrapper blocks like `describe`/`context` should not be deeply nested. As we've discussed before, pytest fixtures handles this in a much nicer way.